### PR TITLE
Updating Az.Resources module dependency to 0.12.0

### PR DESCRIPTION
**Pull request by bganapa**
This is needed as we are releasing a new Az module version aligning with 2108 release
Changes
* Update module versions
* Change module dependencies for az.resources
* PSD1 dependency moved from individual metadata to the top level readme.azurestack.md

### CODE REVIEW

**Review by TheOnlyWei — APPROVED** [review]

File: src/Azs.Backup.Admin/Azs.Backup.Admin.psd1 [hunk 1]
```diff
@@ -0,0 +1,25 @@
+@{
```
Comment (bganapa) [inline src/Azs.Backup.Admin/Azs.Backup.Admin.psd1:1]: Remove this file

File: src/Azs.Backup.Admin/custom/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Custom
```
Comment (bganapa) [inline src/Azs.Backup.Admin/custom/README.md:1]: remove this file

File: src/Azs.Backup.Admin/how-to.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+# How-To
+This document describes how to develop for `Azs.Backup.Admin`.
```
Comment (bganapa) [inline src/Azs.Backup.Admin/how-to.md:2]: remove this file

File: src/Azs.Backup.Admin/license.txt [hunk 4]
```diff
@@ -0,0 +1,227 @@
+MICROSOFT SOFTWARE LICENSE TERMS
+
```
Comment (bganapa) [inline src/Azs.Backup.Admin/license.txt:2]: remove this file

File: src/Azs.Backup.Admin/test/README.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Test
```
Comment (bganapa) [inline src/Azs.Backup.Admin/test/README.md:1]: remove this file

File: src/Azs.Backup.Admin/utils/Unprotect-SecureString.ps1 [hunk 6]
```diff
@@ -0,0 +1,16 @@
+#This script converts securestring to plaintext
```
Comment (bganapa) [inline src/Azs.Backup.Admin/utils/Unprotect-SecureString.ps1:1]: remove this file

**Review by rakku-ms — APPROVED** [review]

**Pull request merged**